testdata/go/v4-alpha/memcached-operator/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.19
 require (
 	github.com/onsi/ginkgo/v2 v2.1.4
 	github.com/onsi/gomega v1.19.0
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.60.1
```

Comment:
@assafad I think we should only add this change and remove changes that are not related to it, unless they are dependencies of 
```
github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.60.1
```